geostore/check_files_checksums/utils.py

```diff
@@ -48,9 +42,10 @@ def log_failure(self, content: JsonObject) -> None:
             LOG_MESSAGE_VALIDATION_COMPLETE, extra={"outcome": Outcome.FAILED, "error": content}
         )
 
-    def validate(self, hash_key: str, range_key: str) -> None:
+    def run(self, hash_key: str, range_key: str) -> None:
```

Comment:
`run` is a very generic name, which I'd normally reserve for methods which do nothing but dispatch, like `main`. This method fetches some asset metadata from DynamoDB and then validates the asset multihash. Maybe a name like `validate_asset_multihash`?

Reply:
I was trying to be generic, because after the changes on following pull request this method will also update processing assets table with 'exist_in_staging' field and 'replaced_in_new_version' field. Both of which don't relate to validation.